src/Files/Dialogs/FilesystemOperationDialog.xaml.cs

```diff
@@ -25,6 +25,15 @@ public FilesystemOperationDialog(FilesystemOperationDialogViewModel viewModel)
 
             ViewModel = viewModel;
             ViewModel.View = this;
+
+            //HACK: Moved setting controls to loaded event seems to fix DefaultButton focus issue.
+            Loaded += (_, _) =>
```

Comment:
This will cause a memory leak, consider registering the event in the UI and having the eventhandler method in code behind :)

Reply:
Ah of course. I’ll get it fixed up.